--- MongoDb.pp.orig	2016-03-17 23:02:12.000000000 +0000
+++ MongoDb.pp	2016-03-17 22:50:24.000000000 +0000
@@ -96,11 +96,11 @@
     })
   }
 
-  if array_true($php, 'install') and ! defined(Puphpet::Php::Pecl['mongo']) {
-    puphpet::php::pecl { 'mongo':
-      service_autorestart => $webserver_restart,
-      require             => Class['mongodb::server']
-    }
-  }
+#  if array_true($php, 'install') and ! defined(Puphpet::Php::Pecl['mongo']) {
+#    puphpet::php::pecl { 'mongo':
+#      service_autorestart => $webserver_restart,
+#      require             => Class['mongodb::server']
+#    }
+#  }
 
 }
